sw/device/silicon_creator/lib/sigverify/sphincsplus/test/BUILD

```diff
@@ -154,10 +158,13 @@ autogen_cryptotest_header(
     tool = ":sphincsplus_set_testvectors",
 )
 
-opentitan_functest(
+opentitan_test(
     name = "verify_test_kat0",
     srcs = ["verify_test.c"],
-    targets = ["cw310_test_rom"],  # Test set is too large for Verilator or ROM
+    exec_env = {
+        # Test set is too large for Verilator or ROM
```

Comment:
What do you mean by "too large for ROM"? Too large to run as a "test in ROM" type test?

Reply:
I simply preserved these comments from the prior rule.

Reply:
After running into an error with another test, I've determined what this comment is meant to convey.

Some of the crypto tests have large data sets that make the test binary too large to be a ROM_EXT stage executed by the real ROM (ie:  greater than 64K).  The test_rom doesn't enforce any of the usual limits, so these tests are fine to be executed in ROM_EXT position by the test_rom.

I've updated the comment to be more clear.